spacy/cli/pretrain.py

```diff
@@ -18,6 +18,7 @@
 from .._ml import Tok2Vec, flatten, chain, create_default_optimizer
 from .._ml import masked_language_model
 from .. import util
+from .train import _load_pretrained_tok2vec
```

Comment:
Move to `util.py`?

Reply:
Probably should, yes. Not strictly necessary for this PR.